src/transformers/convert_graph_to_onnx.py

```diff
@@ -30,14 +16,73 @@ class OnnxConverterArgumentParser(ArgumentParser):
     def __init__(self):
         super(OnnxConverterArgumentParser, self).__init__("ONNX Converter")
 
-        self.add_argument("--pipeline", type=str, choices=SUPPORTED_PIPELINES, default="feature-extraction")
         self.add_argument("--model", type=str, required=True, help="Model's id or path (ex: bert-base-cased)")
         self.add_argument("--tokenizer", type=str, help="Tokenizer's id or path (ex: bert-base-cased)")
         self.add_argument("--framework", type=str, choices=["pt", "tf"], help="Framework for loading the model")
         self.add_argument("--opset", type=int, default=11, help="ONNX opset to use")
         self.add_argument("--check-loading", action="store_true", help="Check ONNX is able to load the model")
         self.add_argument("--use-external-format", action="store_true", help="Allow exporting model >= than 2Gb")
-        self.add_argument("output")
+        self.add_argument("output", type=abspath)
+
+
+if is_torch_available():
+    import torch
+
+    class WrappedModel(torch.nn.Module):
+        def __init__(self, underlying_model):
+            """
+            Class to recursively convert BaseModelOutput* to ONNX supported structs
+            """
+            super(WrappedModel, self).__init__()
+            self.underlying_model = underlying_model
+            self.output_names = {}
+
+        def output_to_tensor(self, output, names=None):
+            if isinstance(output, ModelOutput):
+                parameters = vars(output)
+                output_names = [k for k, v in parameters.items() if v is not None]
+                return self.output_to_tensor(output[:], names=output_names)
+            if isinstance(output, (tuple, list)):
+                output_list = []
+                for o, element in enumerate(output):
+                    element = self.output_to_tensor(element)
+                    # Checks that is a tensor to avoid iterating over it, otherwise flattens the output
+                    if torch.is_tensor(element):
+                        if names is not None:
+                            self.output_names[self.index_output] = names[o]
+                        output_list.append(element)
+                        self.index_output += 1
+                    else:
+                        output_list.extend(element)
+                return output_list
+            return output
+
+        def forward(self, *args, **kwargs):
+            self.index_output = 0
+            output = self.underlying_model(*args, **kwargs)
+            output = self.output_to_tensor(output)
+            return output
```

Comment:
I'm not sure this is needed to support the new kind of named output we introduced recently. 

The easiest and most compatible way would be to just add `return_tuples=True` on the model's config to automagically get the previous behaviour. 

(Tagging @sgugger for awareness)

Reply:
Yes, you can either add `return_tuple=True` (no s) to the call to the `underlying_model` or after the test in `self.output_to_tensor`, you can just do `output = output.to_tuple()`.